test/cri-containerd/container_update_test.go

```diff
@@ -0,0 +1,319 @@
+// +build functional
+
+package cri_containerd
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+type config struct {
```

Comment:
I thought we decided to just make these configs function local from the last time we tried this

Reply:
Like this https://github.com/microsoft/hcsshim/blob/5b2c8a709fadfa327ef90cc8e90051b0919d4bb0/test/cri-containerd/container_network_test.go#L95